packages/blade/src/components/Typography/Heading/Heading.tsx

```diff
@@ -111,13 +115,16 @@ const getProps = <T extends { variant: HeadingVariant }>({
     }
     props.fontSize = 75;
     props.lineHeight = 50;
-    props.as = isPlatformWeb ? 'h6' : undefined;
+    props.as = isPlatformWeb ? 'p' : undefined;
```

Comment:
Was discussing this with saurabh, subheadings as P makes more sense. 

Imagine someone did this: 


```
<Title>Best Payment Gateway Ever</Title>
<Heading variant="subheading">Razorpay</Heading>

<Heading as="h2">10 Ways to Make Free Money</Heading>
```

This will render the HTML as follows

```
<h1>Best Payment Gateway Ever</h1>
<h6>Razorpay</h6>

<h2>10 Ways to Make Free Money</h2>
```

Now ^^ That's wrong, h1 -> h6??? -> h2?? 

The order will be wrong, thus it's best to keep the subheading as plain `p` to avoid invalid semantics. 

> also it might be a breaking change for a lot of products so lets default to as it was?

Won't be any breaking change atleast from visual standpoint, both P and H6 are block level elements.

Reply:
No, making it h6 by default is semantically incorrect behaviour we were shipping by default before. Rendering <p> isn't semantically incorrect. 

Rendering `p` will have normal text content semantic which is better than render h6 because rendering h6 in between other headings will cause [illogical heading order & doesn't create an hierarchical order](https://www.tpgi.com/heading-off-confusion-when-do-headings-fail-wcag/) 

<img width="871" alt="image" src="https://github.com/razorpay/blade/assets/35374649/81a992c3-1a4a-44d8-b96c-3f621b7e2d04">


> now everyone has to always pass as prop always for Title and Heading even

Nope, only for Heading and only if they use variant=subheading. other variants are still as is. And even then it's not an necessary. Because screen reader users doesn't navigate through subheadings they navigate through larger headings and landmarks so rendering p should be okay too.